src/openga.hpp

```diff
@@ -1075,7 +1076,17 @@ class Genetic
 		for(int i=0;i<N;i++)
 			gen.sorted_indices.push_back(i);
 
-		quicksort_indices_SO(gen.sorted_indices,gen,0,int(gen.sorted_indices.size())-1);
+		if (use_quick_search) {
+			quicksort_indices_SO(gen.sorted_indices,gen,0,int(gen.sorted_indices.size())-1);
+		} else {
+			struct {
+				bool operator()(thisGenerationType &gen, int a, int b) const
+				{
+					return gen.chromosomes[a].total_cost < gen.chromosomes[b].total_cost;
+				}
+			} customLess;
```

Comment:
Can be replaced by a lambda function `[const &gen](int a,int b)->{gen.chromosomes[a].total_cost < gen.chromosomes[b].total_cost;}`.

Reply:
fixed